src/pattern-library/components/PlaygroundApp.js

```diff
@@ -97,16 +99,25 @@ export default function PlaygroundApp({
   extraRoutes = [],
 }) {
   const allRoutes = routes.concat(extraRoutes);
-  const [route, navigate] = useRoute(baseURL, allRoutes);
-  const content = route ? (
-    <route.component />
+  const [activeRoute, navigate] = useRoute(baseURL, allRoutes);
```

Comment:
Renamed `route` to `activeRoute` here to avoid confusion as `route` is used for other references.